### DISCUSSION TITLE
Habilitar emissão de CT-e Odoo 16

### DISCUSSION THREAD

**Discussion by hitalodiniz** [OCA/l10n-brazil#3059] [Q&A]
Prezados, 

Instalei o módulo fiscal no Odoo 16, porém não consigo habilitar a emissão de CT-e nas configurações. 

É possível emitir CT-e utilizando a l10n-brazil?

Obrigado.

**Comment by rvalyi** [comment]
Ola @hitalodiniz infelizmente a emissão de CT-e ainda não esta pronta na v16... Na v14 existe um PR experimental que traz esse suporte a emissão de CT-e: https://github.com/OCA/l10n-brazil/pull/2586

Porem falta limpar algumas coisas no modulo spec_driven_model para não trazer conflitos com a emissão de NFe (vou ver se eu consigo fazer isso no mes que vem) e tambem tem que saber que a lib erpbrasil.edoc que pretende fazer a transmissão hoje tem padrões de qualidade muito abaixo da OCA. Ou seja do nosso lado temos mais prioridade em limpar as coisas do que sempre buscar integrar novas funcionalidades...

Se vc tiver experiencia no desenvolvimento, pode seguir esses guidelines se vc pretende limpar e portar essas coisas para a v16:

- https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-15.0
- https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-16.0

Vale a pena dizer que terminar o port do modulo [l10n_br_account para a v16](https://github.com/OCA/l10n-brazil/pull/2865) vai ser importante tb para usar a v16...
